Metro-Records/la-metro-councilmatic
# Add email configuration

## Overview

See title.

Connects #1184 

## Testing Instructions
 
**I did the testing myself, since mine is the only account in the staging database. Works good!**

- Using the review app, go to the Wagtail login, then click Reset password
- Enter your email, then submit the form and confirm you receive a password reset email
